addons/event/report/event_event_templates.xml

```diff
@@ -1,104 +1,241 @@
+                <div class="h-50 col-12 row m-0">
+                    <div class="col-6 h-100 p-0">
+                        <div class="o_event_foldable_badge_step font-weight-bold">3</div>
+                        <img src="/event/static/src/img/how_to_fold_3.png" class="w-100 h-100" alt="How to Fold (3)"/>
+                    </div>
+                    <div class="col-6 h-100 p-0">
+                        <div class="o_event_foldable_badge_step font-weight-bold">4</div>
+                        <img src="/event/static/src/img/how_to_fold_4.png" class="w-100 h-100" alt="How to Fold (4)"/>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<template id="event_registration_report_template_foldable_badge">
+    <t t-call="web.basic_layout">
+        <t t-foreach="docs" t-as="attendee">
+            <t t-set="event" t-value="attendee.event_id.with_context(tz=attendee.event_id.date_tz)"/>
+            <t t-call="event.event_report_template_foldable_badge"/>
         </t>
-    </template>
+    </t>
+</template>
 
+<template id="event_event_report_template_foldable_badge">
+    <t t-call="web.basic_layout">
+        <t t-foreach="docs" t-as="event">
+            <t t-set="event" t-value="event.with_context(tz=event.date_tz)"/>
+            <t t-call="event.event_report_template_foldable_badge"/>
+        </t>
+    </t>
+</template>
 
-    <!-- template use to edit Event Badget (allow to set the HTML badge fields) -->
-    <template id="event_event_report_template_badge">
-        <t t-call="web.basic_layout">
-            <div class="page">
-                <t t-foreach="docs" t-as="event">
-                    <div class="row">
-                        <!-- Front -->
-                        <div class="col-6 text-center" style="padding-left:7mm; border-right:1px dashed black; height: 149mm; padding-top: 7mm">
-                            <div class="row" t-ignore="true">
-                                <div class="col-12">
-                                    <span t-if="event.organizer_id.is_company and event.organizer_id.company_id.logo_web">
-                                        <div t-field="event.event_logo">
-                                            <img t-att-src="image_data_uri(event.organizer_id.company_id.logo_web)" style="max-height:1cm; max-width:4cm;" alt="Logo"/>
-                                        </div>
-                                    </span>
-                                    <h4 t-field="event.name"/>
-                                    <h5>( <i class="fa fa-clock-o" aria-label="Period" role="img" title="Period"></i> <span itemprop="startDate" t-field="event.date_begin" t-options='{"hide_seconds": True}'> </span> <i>to</i> <span itemprop="endDate" t-field="event.date_end" t-options='{"hide_seconds": True}'> </span> )</h5>
+<!-- EVENT FULL PAGE TICKET -->
+
+<template id="event_report_template_full_page_ticket">
+    <div class="row page">
```

Comment:
Such complicated dom ... will not say "clean your dom", but hey can't we try to remove some nested divs ?

Reply:
:it: